interactive_engine/compiler/src/main/java/com/alibaba/graphscope/common/ir/tools/config/PathExpandConfig.java

```diff
@@ -93,6 +97,27 @@ public RelNode getGetV() {
         return getV;
     }
 
+    @Override
+    public String toString() {
+        return "PathExpandConfig{"
```

Comment:
Always use `StringBuilder` for such long string concatenation

Reply:
done